src/storage.rs

```diff
@@ -238,6 +262,8 @@ mod tests {
     // Bytecode from Forge's default Counter.sol contract, compiled with solc 0.8.13.
     // https://github.com/foundry-rs/foundry/blob/nightly-fe2acca4e379793539db80e032d76ffe0110298b/testdata/multi-version/Counter.sol
     const BYTECODE: alloy_primitives::Bytes = bytes!("608060405234801561001057600080fd5b5060f78061001f6000396000f3fe6080604052348015600f57600080fd5b5060043610603c5760003560e01c80633fb5c1cb1460415780638381f58a146053578063d09de08a14606d575b600080fd5b6051604c3660046083565b600055565b005b605b60005481565b60405190815260200160405180910390f35b6051600080549080607c83609b565b9190505550565b600060208284031215609457600080fd5b5035919050565b60006001820160ba57634e487b7160e01b600052601160045260246000fd5b506001019056fea264697066735822122012c25f3d90606133b37330bf079a425dbc650fd21060dee49f715d37d97cb58f64736f6c634300080d0033");
+    const EOF_BYTECODE: alloy_primitives::Bytes =
```

Comment:
Let's add an empty line between the `const`s!

Reply:
Let's also add `Bytes` to the `use alloy_primitives::bytes;` above to make this concise enough for one line. Can you also add a comment on what this bytecode is?